Makefile.am

```diff
@@ -133,50 +118,6 @@ DISTCHECK_CONFIGURE_FLAGS = \
         --with-systemdsystempresetdir=$$dc_install_base/$(systemdsystempresetdir) \
         --with-systemdtmpfilesdir=$$dc_install_base/$(systemdtmpfilesdir)
 
-# copy from autoconf manual, note single suffix rules can't be used there!!
-# NOTE: For non-Makefiles (e.g. for sources), .in conversions should be done in
-# the Makefiles so as to process all the dependencies as sources are updated!
-SED ?= sed
-autopathedit = ${SED} \
-	-e 's|@myDBpath[@]|$(myDBpath)|g' \
-	-e 's|@top_srcdir[@]|$(top_srcdir)|g' \
-	-e 's|@top_builddir[@]|$(top_builddir)|g' \
-	-e 's|@abs_top_srcdir[@]|$(abs_top_srcdir)|g' \
-	-e 's|@abs_top_builddir[@]|$(abs_top_builddir)|g' \
-	-e 's|@mysrcDIR[@]|$(mysrcDIR)|g' \
-	-e 's|@srcdir[@]|$(srcdir)|g' \
-	-e 's|@myDOXDIR[@]|$(myDOXDIR)|g' \
-	-e 's|@mydoxylog[@]|$(mydoxylog)|g' \
-	-e 's|@HAVE_DOXYGEN[@]|$(HAVE_DOXYGEN)|g' \
-	-e 's|@datadir[@]|$(datadir)|g' \
-	-e 's|@datarootdir[@]|$(datarootdir)|g' \
-	-e 's|@PACKAGE[@]|$(PACKAGE)|g' \
-	-e 's|@PACKAGE_NAME[@]|$(PACKAGE_NAME)|g' \
-	-e 's|@PACKAGE_VERSION[@]|$(PACKAGE_VERSION)|g' \
-	-e 's|@prefix[@]|$(prefix)|g' \
-	-e 's|@sysconfdir[@]|$(sysconfdir)|g' \
-	-e 's|@libexecdir[@]|$(libexecdir)|g' \
-	-e 's|@libdir[@]|$(libdir)|g'
```

Comment:
Just in case: did you verify that you moved all of the variables `sed`ed above and actually present in current codebase into the `configuer.ac` script? Because the list in the latter seems notably shorter...

Reply:
I did a make install before and after to make sure the scripts are identical. I guess most of the replacement rules are leftovers from fty-rest.